slack/util.w

```diff
@@ -0,0 +1,15 @@
+bring http;
+
+pub class SlackUtils {
+  pub static inflight post(body: Json, botToken: str) {
```

Comment:
Can't this be a private method of `slack.App`?

Reply:
No since this is needed by context objects `Channel` and `Thread` I wanted to make like an abstract context class that `Channel` and `Thread` can inherit from to get this method, but alas we dont support that.